assets/xml/objects/object_obj_tokeidai.xml

```diff
@@ -1,59 +1,136 @@
+        <Texture Name="gClockTowerSunAndMoonPanelSunTex" OutName="clock_tower_sun_and_moon_panel_sun" Format="rgba16" Width="16" Height="16" Offset="0x4040" />
+
+        <!-- Clock Tower Minute Ring Textures -->
+        <Texture Name="gClockTowerMinuteRingFace1Tex" OutName="clock_tower_minute_ring_face_1" Format="rgba16" Width="32" Height="16" Offset="0x4240" />
+        <Texture Name="gClockTowerMinuteRingFace2Tex" OutName="clock_tower_minute_ring_face_2" Format="rgba16" Width="32" Height="16" Offset="0x4640" />
+        <Texture Name="gClockTowerMinuteRingFace3Tex" OutName="clock_tower_minute_ring_face_3" Format="rgba16" Width="32" Height="16" Offset="0x4A40" />
+        <Texture Name="gClockTowerMinuteRingSidesTex" OutName="clock_tower_minute_ring_sides" Format="rgba16" Width="32" Height="16" Offset="0x4E40" />
+
+        <!-- Clock Tower Clock Face Front Textures -->
+        <Texture Name="gClockTowerClockFaceTopTex" OutName="clock_tower_clock_face_top" Format="ci4" Width="64" Height="64" Offset="0x5240" />
+        <Texture Name="gClockTowerClockFaceBottomTex" OutName="clock_tower_clock_face_bottom" Format="ci4" Width="64" Height="64" Offset="0x5A40" />
+        <Texture Name="gClockTowerClockFacePanelBarTex" OutName="clock_tower_clock_face_panel_bar" Format="rgba16" Width="8" Height="8" Offset="0x6240" />
+
+        <!-- Unused Textures-->
+        <Texture Name="gClockTowerUnusedBrickTex" OutName="clock_tower_unused_bricks" Format="rgba16" Width="32" Height="32" Offset="0x62C0" />
+        <Texture Name="gClockTowerUnusedMaskTex" OutName="clock_tower_unused_mask" Format="i4" Width="64" Height="64" Offset="0x6AC0" />
+
+        <!-- Clock Tower Spotlight Beam Textures -->
+        <Texture Name="gClockTowerSpotlightBeamTex" OutName="clock_tower_spotlight_beam" Format="ia8" Width="32" Height="64" Offset="0x72C0" />
+        <Texture Name="gClockTowerSpotlightMaskTex" OutName="clock_tower_spotlight_mask" Format="i8" Width="32" Height="32" Offset="0x7AC0" />
+
+        <!-- Clock Tower Counterweight Mask Texture -->
+        <Texture Name="gClockTowerCounterweightMaskTex" OutName="clock_tower_counterweight_mask" Format="i4" Width="64" Height="64" Offset="0x7EC0" />
+
+        <!-- Clock Tower Unused Tower Wall Texture -->
+        <Texture Name="gClockTowerUnusedTowerWallMaskTex" OutName="clock_tower_unused_tower_wall_mask" Format="i4" Width="64" Height="64" Offset="0x86C0" />
+
+        <!-- Clock Tower Staircase to Rooftop Texture -->
+        <Texture Name="gClockTowerStaircaseToRooftopMaskTex" OutName="clock_tower_staircase_to_rooftop_mask" Format="i4" Width="64" Height="64" Offset="0x8EC0" />
+
+        <!-- Empty DisplayList, probably intended to be used as the xluDList for Clock Tower Termina Field Walls -->
+        <DList Name="gClockTowerUnusedEmptyDL1" Offset="0x9A00" />
```

Comment:
The type is always the rightmost suffix in extracted assets. This applies to the other dlists too
```suggestion
        <DList Name="gClockTowerUnusedEmpty1DL" Offset="0x9A00" />
```